### PR TITLE
Remove ADV CPR

### PR DESCRIPTION
**Pull request by Tomcat-SG**
**When merged this pull request will:**
- Remove the dependency on Adv CPR and add settings to AED and AED-X
- Remove all known bugs related to kat

### CODE REVIEW

File: addons/circulation/stringtable.xml [hunk 1]
```diff
@@ -312,6 +312,14 @@
...
             <Chinese>使用AED除顫器</Chinese>
 			<Czech>Použít AED defibrilátor</Czech>
+        </Key>
+		<Key ID="STR_KAT_circulation_DefibS_Action_Use">
+            <English>Use AED Station</English>
+            <German>Benutze AED Station</German>
+            <Polish>Użyj stacja AED</Polish>
+            <Spanish>Utilice el estación AED</Spanish>
+            <Chinese>使用AED除顫器</Chinese>
+			<Czech>Použít AED stanice</Czech>
```
Comment (YetheSamartaka) [inline addons/circulation/stringtable.xml]: this should be <Czech>Použít AED stanici</Czech>

Comment (Tomcat-SG) [reply addons/circulation/stringtable.xml]: Is in the new commit

**Comment by Tomcat-SG** [comment]
All Animations need to be tested

**Comment by Tomcat-SG** [comment]
We need only a solution for the animations and after that, this update is done

**Pull request merged**